app/utils/ipfs.js

```diff
@@ -14,3 +14,8 @@ export async function getText(hash) {
   const getResult = await getIpfsApi().get(hash);
   return getResult[0].content.toString('utf8');
 }
+
+export async function getImage(hash) {
```

Comment:
Can we just manually constract image URL without calling method get? Does get actually return image bytes?

Reply:
we store image hash in ipfs object, this method returns image data through image hash. then we get image url through new Blob